pupil-spa/src/app/services/submission/submission.service.spec.ts

```diff
@@ -32,6 +32,20 @@ describe('SubmissionService', () => {
     expect(service).toBeTruthy();
   }));
 
+  it('submitCheckStartData function should call storageService getItem', inject([SubmissionService], (service: SubmissionService) => {
+    mockBackend.connections.subscribe((connection) => {
+      connection.mockRespond(new Response(new ResponseOptions({
+        body: JSON.stringify({isSaved: true}),
+        status: 200
+      })));
+    });
+    spyOn(storageService , 'getItem').and.returnValues({});
+    submissionService.submitCheckStartData().then(() => {
+      expect(storageService.getItem).toHaveBeenCalledWith('pupil');
+      expect(storageService.getItem).toHaveBeenCalledWith('access_token');
```

Comment:
FYI: we will be moving the JWT token to http-only very soon.  This means it will be sent with each request by the browser, not as part of the body.

Reply:
@GuyHarwood ok that's better, we'll definitely tackle this at that stage.